tests/performance/NiO/CMakeLists.txt

```
@@ -22,8 +22,15 @@ function(
   endif()
 
   set(PPDIR "${qmcpack_SOURCE_DIR}/tests/performance/NiO")
-
-  execute_process(COMMAND ${CMAKE_COMMAND} -E copy_directory ${SDIR} ${WDIR})
+  set(PNBDIR "${qmcpack_BINARY_DIR}/tests/performance/NiO/")
+
+  #  execute_process(COMMAND ${CMAKE_COMMAND} -E copy_directory ${SDIR} ${WDIR})
+  execute_process(COMMAND ${CMAKE_COMMAND} -E make_directory ${WDIR})
```

```suggestion
  file(MAKE_DIRECTORY ${WDIR})
```
https://cmake.org/cmake/help/latest/command/file.html#make-directory